cmd/kubeadm/app/constants/constants.go

```diff
@@ -155,6 +155,12 @@ const (
 	// after https://github.com/kubernetes/kubernetes/pull/53833 being merged.
 	KubeletBaseConfigurationConfigMapKey = "kubelet"
 
+	// KubeletBaseConfigurationDir specifies the directory on the node where stores the initial remote configuration of kubelet
+	KubeletBaseConfigurationDir = "/var/lib/kubelet/config/init"
+
+	// KubeletBaseConfigurationFile specifies the file name on the node which stores initial remote configuration of kubelet
+	KubeletBaseConfigurationFile = "kubelet.yaml"
```

Comment:
This constant is defined somewhere in the kubelet codebase. Right now it is `kubelet` I think. You can check the PR where @mtaufen is in process to graduate file-loading from alpha to beta

Reply:
OK, I will check it. Thanks for pointing it out!

Reply:
Updated. I add a TODO here.

https://github.com/kubernetes/kubernetes/pull/53833/files#diff-fbd96764b3922031c77bb5efdab3269aR21